tests/config/test_config.py

```diff
@@ -0,0 +1,28 @@
+# pylint: disable=missing-function-docstring disable=missing-module-docstring
+
+from nestor_api.config.config import Configuration
+
+
+def test_get_config_path(monkeypatch):
+    monkeypatch.setenv("NESTOR_CONFIG_PATH", "/a-custom-config-path")
+    assert Configuration.get_config_path() == "/a-custom-config-path"
```

Comment:
No `test_get_config_path_configured` ?

Reply:
It always needs to be configured, there is no default for this one, so there is only one test 🙂